.github/workflows/benchmark.yml

```diff
@@ -138,6 +157,15 @@ jobs:
           command: |
             kubectl logs -n ${{ steps.setup.outputs.namespace }} dragonfly-sample-1
 
+      - name: Get kubectl logs
```

Comment:
maybe update this as describe dragonflydb object as we only do that here and nothing about logs?